include/mega/node.h

```diff
@@ -262,9 +262,10 @@ struct MEGA_API Node : public NodeCore, FileFingerprint
     ~Node();
 
 private:
+#ifdef ENABLE_SYNC
     // whether this node can be synced to the local tree
     bool mSyncable = true;
```

Comment:
This is no longer used. And ENABLE_SYNC macros should be moved to LocalNode instead. And control deserialization in that case, to skip the corresponding byte without assigning its value

Reply:
It is still used, indeed, by the `Node::setSyncable()` and `Node::isSyncable()`, which I guess Christian will eventually use for the one-way-sync.

Reply:
Furthermore, the `LocalNode::mSyncable` is already between the macros, as it is the full `LocalNode` declaration :)